README.md

```diff
@@ -39,6 +39,22 @@ Other connection mode values are:
 * ovs-bridge - same as a regular bridge, but uses OvS. Can pass LACP traffic.
 * macvtap
 
+## Management interface
```

Comment:
@kaelemc would you be able to try this out on some cisco devices that you're familiar with?

we've added SR OS support and people are trying it right now, but so far it looks very positive. vJunos has been tested by @vista- and some other platforms we will test in the community stream

Reply:
Yeah, sure thing 👍

Reply:
So with IOS/IOS-XE (XR is fine), we have to supply the subnet mask for v4 addresses in dotted decimal (think `255.255.255.0`) since the CIDR prefix length is embedded in `mgmt_address_ipv4`.

Should I just use the `/` as a delimiter and convert the prefix length into the DDN mask? (for IOL we do a similar conversion in [containerlab](https://github.com/srl-labs/containerlab/blob/13f951f6d0d5244a5e029b4227b49071b5b6eccc/utils/ip.go#L35C1-L47C2))

Reply:
Yes, that's fine